__tests__/app.test.js

```diff
@@ -12,6 +12,41 @@ beforeEach(() => {
     return seed(data)
 })
 
+describe("GET/POST/PATCH/DELETE /api/* - (Global Errors)", ()=>{
```

Comment:
There only needs to be one test here. Take a look at your tests, they're all doing the same thing! 😄 Whilst I can see you've got a separate test to make sure it'll be picked up by each method, it isn't necessary. The 404 handler will pick up anything where an invalid path has been entered 👍